CHANGELOG.md

```diff
@@ -3,6 +3,25 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.2 - Gray Eagle [11/01/2023]
+
+### `Added`
+
+- Update module pbccs (v6.4.0)
+- Update module isoseq refine (v3.8.1)
+- Update module lima (v2.6.0)
```

Comment:
This looks better and is more informative in a table format, e.g. as in https://github.com/nf-core/ampliseq/blob/master/CHANGELOG.md#dependencies-1, imho. Just as a comment.

Reply:
Thanks for this suggestion, already I added it.